arch/arm/src/sama5/sam_udphs.c

```diff
@@ -2439,7 +2434,12 @@ static void sam_dma_interrupt(struct sam_usbdev_s *priv, int epno)
         }
       else if (privep->epstate == UDPHS_EPSTATE_RECEIVING)
         {
-          /* privreg->inflight holds the total transfer size */
+          /* privreg->inflight holds the total transfer size
+           * Get the request from the head of the endpoint request queue
+          */
+
+          privreq           = sam_rqpeek(&privep->reqq);
```

Comment:
why add so many spaces before =

Reply:
I was matching the style of the following code. Happy to change if preferred?